roles/cloud_connector/tasks/main.yml

```diff
@@ -58,5 +58,9 @@
   vars:
     client_id: "{{ cert_output.stdout.replace('subject= /CN=', '') }}"
 
+- name: Configure HTTP proxy
```

Comment:
At this step, rhcd is already running and nothing restarts it.

Also, nothing triggers a systemd daemon reload, so the dropin is not loaded either.